scripts/run-mod-settings.sh

```diff
@@ -0,0 +1,184 @@
+#!/bin/bash
+
+set -e
+U=http://localhost:9130
+T=testlib15
+username=testing_admin
+password=admin
+curl -d"{\"id\":\"$T\"}" $U/_/proxy/tenants
+curl -d'{"name":"DB_HOST","value":"localhost"}' $U/_/env
+curl -d'{"name":"DB_PORT","value":"5432"}' $U/_/env
+curl -d'{"name":"DB_USERNAME","value":"postgres"}' $U/_/env
+curl -d'{"name":"DB_PASSWORD","value":"postgres3636"}' $U/_/env
+curl -d'{"name":"DB_DATABASE","value":"postgres"}' $U/_/env
+
+# Set of modules that are necessary to bootstrap admin user
+CORE_MODULES="mod-users mod-login mod-permissions"
```

Comment:
We don't need authtoken?

Reply:
Mod-authtoken is the one enabled after core.

Reply:
Oh yes, of course.